src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -84,6 +84,14 @@ public function getConfigTreeBuilder()
                                 'base_urls' => array_values(array_unique(array_merge($config['base_urls']['http'], $config['base_urls']['ssl']))),
                             );
                         }
+                    } else {
```

Comment:
Answering my own question: not necessarily. 

We need to enable assets if they're not enabled, but only if templating doesn't contain asset configuration.

PR Updated.